### ISSUE TITLE
concrete simulation's model window resizing 

### ISSUE THREAD

**Issue by EstherHKim** [UPPAALModelChecker/UPPAAL-Meta#97]
**Is your feature request related to a problem? Please describe.**
It is not a problem more suggestion for idea. When I am using the concrete simulation, I can see that the each model's window is fixed. And when there are lots of different size of windows, it is not looks aligned. 

**Describe the solution you'd like**
I hope I can resize the window's size. 

**Describe alternatives you've considered**
no.

**Additional context**
![UPPAAL_expect_01](https://user-images.githubusercontent.com/89918210/161762679-69dd6a20-9779-45c8-9421-3a248bd5f4c0.png)

**Comment by RasmusRendal** [comment]
v4.1.20-stratego-8 added support for zooming in the simulator like so, by using the scroll wheel and holding down control, or by using the buttons in the toolbar. Is that what you mean?

https://user-images.githubusercontent.com/19538856/161763782-e6551220-ebb2-48ad-b39c-0268904925b0.mp4

**Comment by EstherHKim** [comment]
No, not the whole window of the concrete simulation. I mean **each one** of the timed automaton size control. For example, your torch and viling1 window size is different. And I want to make it same, bigger or smaller. But this is just suggestion for detail gui feature.  :)
